Packs/Core/Playbooks/playbook-Remote_WMI_Process_Execution.yml

```diff
@@ -0,0 +1,1004 @@
+id: Remote WMI Process Execution
+version: -1
+name: Remote WMI Process Execution
+description: |-
```

Comment:
Consider changing the description format to match that of the other playbook we released.